etc/sysconfig/pacemaker.in

```diff
@@ -81,6 +81,12 @@
 # Default: PCMK_debug="no"
 # Example: PCMK_debug="pacemakerd,pacemaker-execd"
 
+# PCMK_stderr (Advanced Use Only)
+#
+# Whether to send daemon log messages to stderr.
```

Comment:
Should it be documented in sysconfig or not? I don't think sysconfig will be read if a user starts Pacemaker manually on the command line.

Reply:
I guess it could go in there as preparation for creating a "true" config file (T574). In the future, we'll likely parse the configuration file in a more sophisticated way than passing it to the shell to pull environment variables.

Reply:
Yeah, it's worth it just to have the documentation. Probably worth adding to the comment that setting it in this file is pointless.